crates/ruma-state-res/src/lib.rs

```diff
@@ -238,28 +238,28 @@ fn reverse_topological_power_sort<E: Event>(
 
     lexicographical_topological_sort(&graph, |event_id| {
         let ev = fetch_event(event_id).ok_or_else(|| Error::NotFound("".into()))?;
-        let pl = event_to_pl.get(event_id).ok_or_else(|| Error::NotFound("".into()))?;
-
-        debug!("{:?}", (-*pl, ev.origin_server_ts(), &ev.event_id()));
-
-        // This return value is the key used for sorting events,
-        // events are then sorted by power level, time,
-        // and lexically by event_id.
```

Comment:
I think this comment is still relevant, at least in some form.

Reply:
But it's basically the same as on the function being called, no?

Reply:
…true, then I think this is alright.